scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -1,131 +1,142 @@
 {% macro institute_settings(form, beacon_form, institute, current_user, loqus_instances) %}
-<div class="container">
-
```

Comment:
These might look a lot of changes, but I've just re-arranged and improved the looks of the page, from this:

<img width="1436" alt="image" src="https://user-images.githubusercontent.com/28093618/186167564-4648e103-10fb-47fb-9cab-750c4ea660a8.png">


To this:

<img width="1433" alt="image" src="https://user-images.githubusercontent.com/28093618/186167362-ee566556-f2cb-42ff-b778-2dcd49f9cfdc.png">